addon/components/power-calendar/days.js

```diff
@@ -79,9 +84,6 @@ export default Component.extend({
     let i = 0;
     while (days[i]) {
       let daysOfWeek = days.slice(i, i + 7);
-      if (!showDaysAround) {
```

Comment:
handled in template instead; allows for better screen layout in the `false` state of this condition